verbose_debugging_reports.md

```diff
@@ -29,6 +29,9 @@ is rejected due to the following limits to mitigate security concerns:
 #### `source-destination-rate-limit`
 A source is rejected due to the [destinations per source and reporting site rate limit](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#limiting-the-number-of-unique-destinations-per-source-site).
 
+#### `source-destination-wider-rate-limit`
```

Comment:
IMO `wider` is not descriptive enough, and IIUC the value in question isn't even necessarily wider than any other limit, since these are vendor-specific values.

CC @johnivdel for thoughts on naming.

Reply:
Yeah I agree that `wider` may not be good, alternatively we can change this to per day and also make the new rate limit window constant.

Reply:
Gave it a try, hopefully this is clearer.